torch/onnx/__init__.py

```diff
@@ -90,6 +91,9 @@
     symbolic_opset18,
     symbolic_opset19,
     symbolic_opset20,
+    symbolic_opset21,
+    symbolic_opset22,
+    symbolic_opset23,
```

Comment:
We should not expose any new modules to public. They only exist for backward compatibility reasons. Prefer reverting changes to this file